_maps/interiors/apc_medical.dmm

```diff
@@ -0,0 +1,173 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/closed/interior/apc/nineteen,
+/area/interior/apc)
+"b" = (
+/obj/structure/bed/chair/driver_gunner_seat{
+	pixel_x = -3;
+	pixel_y = -5
+	},
+/turf/open/interior/apc/eleven,
+/area/interior/apc)
+"c" = (
+/turf/open/interior/apc/nine,
+/area/interior/apc)
+"d" = (
+/turf/closed/interior/apc/twenty,
+/area/interior/apc)
+"e" = (
+/turf/closed/interior/apc/two,
+/area/interior/apc)
+"g" = (
+/turf/closed/interior/apc/five,
+/area/interior/apc)
+"h" = (
+/turf/closed/interior/apc/twelve,
+/area/interior/apc)
+"k" = (
+/obj/machinery/optable,
+/obj/item/tank/anesthetic,
+/turf/open/interior/apc/twentynine,
+/area/interior/apc)
+"l" = (
+/turf/closed/interior/apc/twentyseven,
+/area/interior/apc)
+"m" = (
+/obj/structure/gun_breech/secondary,
+/turf/open/interior/apc/twentytwo,
+/area/interior/apc)
+"n" = (
+/turf/open/interior/apc/ten,
+/area/interior/apc)
+"o" = (
+/turf/open/interior/apc/sixteen,
+/area/interior/apc)
+"p" = (
+/turf/closed/interior/apc/thirteen,
+/area/interior/apc)
+"s" = (
+/turf/open/interior/apc/eight,
+/area/interior/apc)
+"t" = (
+/obj/structure/bed/chair/dropship/doublewide/left,
+/obj/structure/bed/chair/dropship/doublewide/right,
+/turf/open/interior/apc/nine,
+/area/interior/apc)
+"B" = (
+/turf/closed/interior/apc/six,
+/area/interior/apc)
+"C" = (
+/turf/closed/interior/tank/door,
+/area/interior/apc)
+"D" = (
+/turf/open/interior/apc/fifteen,
+/area/interior/apc)
+"F" = (
+/obj/machinery/vending/MarineMed,
```

Comment:
wasn't being able to deploy a vendor with infinite meds something we didn't want?

Reply:
Kuro said so blame him